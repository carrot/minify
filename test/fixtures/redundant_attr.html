<form method='get' wow='amaze'></form>
<input type='text' wow='amaze'>
<button type='submit' wow='amaze'></button>
<link media='all' wow='amaze'>
<style media='all' type='text/css' wow='amaze'></style>
<script type='text/javascript' language='javascript' wow='amaze'></script>
<area shape='rect' wow='amaze'>
