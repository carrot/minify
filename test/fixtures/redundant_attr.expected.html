<form wow="amaze"></form><input wow="amaze"><button wow="amaze"></button><link wow="amaze"><style wow="amaze"></style><script wow="amaze"></script><area wow="amaze">
